src/main/java/ti4/helpers/async/RoundSummaryHelper.java

```diff
@@ -66,21 +68,31 @@ public static void finishEditRoundSummary(ModalInteractionEvent event, Game game
         if (matcher.matches()) {
             ModalMapping mapping = event.getValue("summary");
             String thoughts = mapping.getAsString();
-            storeEndOfRoundSummary(game, player, matcher.group("round"), thoughts, false);
+            storeEndOfRoundSummary(game, player, matcher.group("round"), thoughts, false, event.getChannel());
         }
     }
 
-    public static void storeEndOfRoundSummary(Game game, Player player, String roundNum, String thoughts, boolean append) {
+    public static void storeEndOfRoundSummary(Game game, Player player, String roundNum, String thoughts, boolean append, MessageChannel eventChannel) {
         roundNum = roundNum.replaceAll("[^0-9]", ""); // I only want the digits
-        String roundKey = "endofround" + roundNum + player.getFaction();
+        String roundKey = resolveRoundSummaryKey(player, roundNum);
         String previousThoughts = "";
         if (append && !game.getStoredValue(roundKey).isEmpty()) {
             previousThoughts = game.getStoredValue(roundKey);
             previousThoughts = previousThoughts.replaceFirst("\\.? ?$", "") + "\n";
         }
         game.setStoredValue(roundKey, previousThoughts + thoughts);
 
-        MessageHelper.sendMessageToChannelWithButton(player.getCardsInfoThread(), player.getFactionEmoji() + " stored an end of round summary", Buttons.EDIT_SUMMARIES);
+        MessageChannel playerChannel = player.isRealPlayer() ? player.getCardsInfoThread() : eventChannel;
+        MessageHelper.sendMessageToChannelWithButton(playerChannel, resolvePlayerEmoji(player) + " stored an end of round summary", Buttons.EDIT_SUMMARIES);
         MessageHelper.sendMessageToChannel(game.getMainGameChannel(), "Someone stored an end of round summary");
     }
+
+    public static String resolveRoundSummaryKey(Player player, String roundNum) {
+        String keySuffix = player.isRealPlayer() ? player.getFaction() : player.getUserName();
```

Comment:
can you check if an eliminated player would get their end of round summary (after elimination) would get posted as a separate good doggo entry?

Reply:
Will do. I have time tomorrow. :)